chips/src/main/java/com/decathlon/vitamin/compose/chips/impl/VitaminInputChipImpl.kt

```diff
@@ -0,0 +1,125 @@
+package com.decathlon.vitamin.compose.chips
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.FilterChip
+import androidx.compose.material.LocalContentAlpha
+import androidx.compose.material.Text
+import androidx.compose.material.ripple.LocalRippleTheme
+import androidx.compose.material.ripple.RippleTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.draw.clip
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+import com.decathlon.vitamin.compose.foundation.VtmnStatesDisabled
+
+@Suppress("LongMethod")
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+internal fun VitaminFilterChipImpl(
```

Comment:
The filename isn't `VitaminFilterChipImpl`

Reply:
oh yes, I changed it